### DISCUSSION TITLE
Open Educational Resource (in French) published using Quarto!

### DISCUSSION THREAD

**Discussion by fuhrmanator** [quarto-dev/quarto-cli#4860] [Show and Tell]
Hello,

I'm proud and happy to share that since March 1st, it's [official](https://etsmtl.on.worldcat.org/oclc/1371977982): the book for the course _LOG210 Analyse et conception de logiciels_ (Software analysis and design) was "published" as an Open Educational Resource (OER, Ressource éducative libre REL in French) and we used Quarto to make it. 🚀 

I want to thank the Quarto team for helping along the way - more than 10 bugs were fixed and more than 20 discussions started by me here were handled. It was very encouraging to have such a quick turn-around from the team, even if sometimes the output broke while running on the bleeding edge (1.3). It's a great example of how open source works well.

**Comment by fuhrmanator** [comment]
I'm attaching some slides (reveal.js, of course made with Quarto) I presented at the [OER showcase for Open Education week at the University of Ottawa](https://www.uottawa.ca/library/open-educational-resources/open-education-week) last week. 
[open-source-tools-fabriqueREL2023.zip](https://github.com/quarto-dev/quarto-cli/files/10995926/open-source-tools-fabriqueREL2023.zip)

**Comment by cderv** [comment]
That is cool ! I have put a link in the gallery : https://quarto.org/docs/gallery/

Thanks for sharing !

**Comment by dragonstyle** [comment]
Congratulations and thank you for the work you put into using Quarto and helping us diagnose issues! We made some great improvements to our PDF rendering in particular (among other areas too!) that I can think of thanks to your efforts - we appreciate it!
